tables/zenodo-upload/README.md

```diff
@@ -51,120 +51,120 @@ Where applicable, all tables are ordered by either `sample_id` or `Kids_First_Bi
 |--------|-------------|--------------------|-------------------------|
-|   5A     |  | UMAP of tumors colored by broad histology           |
-|   5B     |  | Heatmap of GSVA scores across cancer groups           |
-|   5C     |  | Box/strip plots of quanTIseq scores across cancer groups           |
-|   5D     |  | Forest plot from survival analysis exploring CD274 expression and immune cell proportion effects           |
+|   5A     | `figure-5a-data.csv`  | UMAP of tumors colored by broad histology           |
+|   5B     | `figure-5b-data.csv`  | Heatmap of GSVA scores across cancer groups           |
+|   5C     | `figure-5c-data.csv`  | Box/strip plots of quanTIseq scores across cancer groups           |
+|   5D     | `figure-5d-data.csv`  | Forest plot from survival analysis exploring CD274 expression and immune cell proportion effects           |
 
 
 ### Supplemental Figure Data
 
 #### Figure S2
 
-| Figure Panel | Filename | Brief Description | Notes on Tabular Data | 
+| Figure Panel | Filename | Brief Description | Notes on Tabular Data |
 |--------|-------------|--------------------|-------------------------|
-|   S2A     |             |             |
-|   S2B     |             |             |
-|   S2C     |             |             |
-|   S3D     |             |             |
-|   S2E     |             |             |
-|   S2F     |             |             |
-|   S2G     |             |             |
-|   S2H     |             |             |
-|   S2I     |             |             |
+|   S2A     |  `figure-S2a-S2b-S2c-data.csv`   | Correlations of mutation VAFs from variant callers for PBTA data   | This compressed file contains data for all three panels S2A, S2B, and S2C. Caution: Uncompressed, this file is ~3.9 GB.|
+|   S2B     |  `figure-S2a-S2b-S2c-data.csv`  | Distributions of mutation VAFs from variant callers for PBTA data  | This compressed file contains data for all three panels S2A, S2B, and S2C. Caution: Uncompressed, this file is ~3.9 GB.   |
+|   S2C     |  `figure-S2a-S2b-S2c-data.csv`  | Upset plot of overlap among variant callers for PBTA data  | This compressed file contains data for all three panels S2A, S2B, and S2C. Caution: Uncompressed, this file is ~3.9 GB.  |
+|   S3D     |   `figure-S2d-S2e-S2f-data.csv`    |  Correlations of mutation VAFs from variant callers for TCGA data       | This compressed file contains data for all three panels S2D, S2E, and S2F. Uncompressed, this file is 13MB.            |
+|   S2E     |   `figure-S2d-S2e-S2f-data.csv`  |  Distributions of mutation VAFs from variant callers for PBTA data         |   This compressed file contains data for all three panels S2D, S2E, and S2F. Uncompressed, this file is 13MB.           |
+|   S2F     |   `figure-S2d-S2e-S2f-data.csv`  | Upset plot of overlap among variant callers for TCGA data    |  This compressed file contains data for all three panels S2D, S2E, and S2F. Uncompressed, this file is 13MB.            |
```

Comment:
Should we report compression here in the filenames (e.g., `figure-S2a-S2b-S2c-data.csv.gz`)? (I haven't looked at the upload itself at this point.)

Reply:
I wonder if we might condense this:

```suggestion
|   S2A-C    |  `figure-S2a-S2b-S2c-data.csv`   | Correlations, distributions, and UpSet plot of mutation VAFs from variant callers for PBTA data, respectively   | This compressed file contains data for all three panels S2A, S2B, and S2C. Caution: Uncompressed, this file is ~3.9 GB.|
|   S3D-F     |   `figure-S2d-S2e-S2f-data.csv`    |  Correlations, distributions, and UpSet of mutation VAFs from variant callers for TCGA data, respectively       | This compressed file contains data for all three panels S2D, S2E, and S2F. Uncompressed, this file is 13MB.            |
```

Side note - the figure legend in the paper should probably be stylized as "UpSet" also ([Wikipedia](https://en.wikipedia.org/wiki/UpSet_Plot)).